### PR TITLE
Ocular gatsby website

### PR DESCRIPTION
**Pull request by jckr**
This is a port of the website with current ocular-gatsby.

### CODE REVIEW

File: website/.gitignore [hunk 1]
```diff
@@ -0,0 +1,59 @@
+### Node ###
```
Comment (ibgreen) [inline website/.gitignore]: Nit: Where is this from? This file seems way too big many of these lines are completely irrelevant. 

No real harm but rather not confuse users with unrelated stuff.

File: website/ocular-config.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  THEME_OVERRIDES: [
+    //  {key: 'primaryFontFamily', value: 'serif'}
+  ],
+
+  PROJECT_TYPE: 'github',
+  PROJECT_NAME: 'viewport-mercator-project',
+  PROJECT_ORG: 'uber-common',
+  PROJECT_URL: 'https://github.com/uber-common/viewport-mercator-project',
+  PROJECT_DESC: 'Utilities for working with Web Mercator Projections',
+  PATH_PREFIX: '',
```
Comment (ibgreen) [inline website/ocular-config.js]: We do need a path prefix right?

File: website/package.json [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
-  "name": "viewport-mercator-project-website",
+  "name": "viewport-mercator-project",
+  "version": "1.0.0",
+  "description": "Utilities for working with Web Mercator Projections",
   "main": "index.js",
   "scripts": {
-    "start": "ocular start open",
-    "build": "ocular build"
+    "start": "yarn clean && yarn develop",
+    "build": "yarn clean && yarn clean-public && gatsby build",
```
Comment (ibgreen) [inline website/package.json]: We need the --prefix option here.

File: website/package.json [hunk 4]
```diff
@@ -1,14 +1,24 @@
...
+    "clean": "rm -rf ./.cache",
+    "clean-public": "rm -rf ./public",
+    "develop": "gatsby develop --port=801",
```
Comment (ibgreen) [inline website/package.json]: Drop the port number?

**Comment by jckr** [comment]
Screen Shot 2019-09-03 at 7 40 46 PM

File: website/package.json [hunk 5]
```diff
@@ -1,14 +1,24 @@
...
+    "serve": "gatsby serve",
+    "deploy": "NODE_DEBUG=gh-pages gh-pages -d public"
   },
   "keywords": [],
   "author": "",
-  "license": "MIT",
+  "license": "ISC",
```
Comment (ibgreen) [inline website/package.json]: Why did we change this? Assuming it was a mistake.

File: website/.gitignore (hunk 1 above)
Comment (jckr) [reply website/.gitignore]: for this and for your other comment on license - 

my process was:
- npm init -y (which set the license to ISC)
- yarn add ocular-gastby@latest gatsby@latest react@latest react-dom@latest
- ocular init
- some touch-ups of ocular-config.js and package.json based on loaders.gl

i suspect the .gitignore is generated by npm init.

File: website/ocular-config.js (hunk 2 above)
Comment (jckr) [reply website/ocular-config.js]: good call. thanks
